scrapy/downloadermiddlewares/retry.py

```diff
@@ -31,6 +31,105 @@
 logger = logging.getLogger(__name__)
 
 
+def get_retry_request(
+    request,
+    *,
+    reason,
```

Comment:
what about adding a default reason? (i.e.: `reason='retry'` or `'default_retry'` or something similar...) in that way you don't need to give a name if you don't need to differentiate them.

Reply:
`unspecified`?

Reply:
it could work :+1: